### PR TITLE
Fix `git maintenance` invocation

### PR DESCRIPTION
**Pull request by ominitay**
Before submitting your PR (pull request), please
check the following:
* [x] There is no other PR (open or closed)
  similar to yours. If there is, please first
  discuss over there.
* [x] Each commit messages follows the [Seven
  Rules of a Great Commit
  Message](https://cbea.ms/git-commit/#seven-rules).
* [x] Each commit message includes
  `Fixes #<bug>` or `Resolves #<issue>` in its 
  body (not subject) for each issue it resolves
  (if any).
* [x] You have
  [squashed](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing)
  any redundant or insignificant commits.

Currently, `znap uninstall` fails, along with disabling `git maintenance` with `znap pull` because of this bug.

Related: #140

### CODE REVIEW

File: functions/.znap.uninstall [hunk 1]
```diff
@@ -18,7 +18,7 @@ private repo=
 for repo in ~znap/$^@:t; do
   if [[ -d $repo ]]; then
     is-at-least 2.31 ${${=$( git --version )}[3]} &&
-        git -C $repo maintenance unregister --quiet
+        git -C $repo maintenance unregister $> /dev/null
```
Comment (marlonrichert) [inline functions/.znap.uninstall:21]: You appear to have a typo here.

Comment (marlonrichert) [inline functions/.znap.uninstall:21]: You appear to have a typo here:

```suggestion
        git -C $repo maintenance unregister &> /dev/null
```

Comment (ominitay) [reply functions/.znap.uninstall:21]: Thanks for catching that :)

Comment (marlonrichert) [reply functions/.znap.uninstall:21]: Once you have fixed that, I will merge your commit.

**Comment by ominitay** [comment]
Apologies for taking a while to get that fixed :)

**Comment by marlonrichert** [comment]
NP :)

**Pull request merged**